exporters/stdout/stdoutmetric/config.go

```diff
@@ -74,6 +81,22 @@ func WithEncoder(encoder Encoder) Option {
 	})
 }
 
+// WithWriter sets the export stream destination.
+// Using this option overrides any previously set encoder.
+func WithWriter(w io.Writer) Option {
+	return WithEncoder(json.NewEncoder(w))
+}
+
+// WithPrettyPrint prettifies the emitted output.
+// This option only works if the encoder is a *json.Encoder, as is the case
+// when using `WithWriter`.
```

Comment:
Can we also fix the comments in `stdouttrace`? It can be in a seperate followup PR if you prefer.

Reply:
stdouttrace works differently, because it doesn't have `WithEncoder`. So the encoder is always a JSON one.
Similarly, since it doesn't have a `WithEncoder`, `WithWriter` can't be overriden.

Reply:
PTAL https://github.com/open-telemetry/opentelemetry-go/pull/4544